packages/api/src/routes/internal.ts

```diff
@@ -304,6 +304,7 @@ router.get(
  * @param req.query.cxId - The cutomer's ID.
  * @param req.query.cwEnabled - Whether to enabled CommonWell.
  * @param req.query.cqEnabled - Whether to enabled CareQuality.
+ * @param req.query.epicEnabled - Whether to enabled CareQuality.
```

Comment:
nit: missed updating `CareQuality` to `Epic`

Reply:
Done.